src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -203,6 +203,8 @@ public function convert(FileVersion $fileVersion, $formatKey, $imageFormat)
             }
         } catch (RuntimeException $exception) {
             // ignore exceptions here (some imagine adapter does not implement this)
+        } catch (\RuntimeException $exception) {
```

Comment:
Wasn't the reason we were waiting for the other merge, that we don't need that `catch` anymore?